tests/integration/security/util/config/config.go

```diff
@@ -0,0 +1,50 @@
+//go:build integ
+// +build integ
+
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/util/file"
+	"istio.io/istio/pkg/test/util/tmpl"
+)
+
+type ApplyFn func(t framework.TestContext, ns string, args map[string]string)
+
+func None() ApplyFn {
+	return func(framework.TestContext, string, map[string]string) {}
+}
+
+func NoWait(filePaths ...string) ApplyFn {
+	return func(t framework.TestContext, ns string, args map[string]string) {
+		t.Helper()
+		policy := tmpl.EvaluateAllOrFail(t, args, file.AsStringArrayOrFail(t, filePaths...)...)
+		t.ConfigIstio().ApplyYAMLOrFail(t, ns, policy...)
+	}
+}
+
+func WithWait(filePaths ...string) ApplyFn {
+	return func(t framework.TestContext, ns string, args map[string]string) {
+		t.Helper()
+		policy := tmpl.EvaluateAllOrFail(t, args, file.AsStringArrayOrFail(t, filePaths...)...)
+		// Have to apply/wait one at a time.
```

Comment:
can we not apply all then wait for all?

since if you apply multiple quickly pilot will batch them